Samples/src/VisualDebugger/Hubs.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.SignalR;
+using System.Threading.Tasks;
+
+namespace vis_sim
```

Comment:
We should probably put this sample under the `Microsoft.Quantum.Samples.VisualDebugger` namespace...